src/en/mangatop/src/eu/kanade/tachiyomi/extension/en/mangatop/MangaScans.kt

```diff
@@ -28,16 +28,18 @@ import java.text.SimpleDateFormat
 import java.util.Calendar
 import java.util.Locale
 
-class MangaTop : ParsedHttpSource() {
+class MangaScans : ParsedHttpSource() {
 
-    override val name = "MangaTop"
+    override val name = "MangaScans"
 
-    override val baseUrl = "https://mangatop.to"
+    override val baseUrl = "https://mangascans.to"
 
     override val lang = "en"
 
     override val supportsLatest = true
 
+    override val id = 85127596998931837
```

Comment:
I'm sorry if I'm bothering you, but I've been meaning to ask you for a while, how do I get this ID?

Reply:
https://github.com/keiyoushi/extensions/blob/repo/index.json

Reply:
> https://github.com/keiyoushi/extensions/blob/repo/index.json

Ohh there it turns out, thank you for answering my question